### DISCUSSION TITLE
回望与兔年寄语

### DISCUSSION THREAD

**Discussion by ShixiangWang** [openbiox/weekly#1328] [Announcements]
大家发表感想和想法。

**Comment by He-Kai-fly** [comment]
## 回望 
加入Openibox社区已半年有余，很开心能分配到诗翔带领的生信爱好者周刊小分队中，刚进入团队时，从内容搜集到周日发表整个流程都已规划的十分详细了，使得大家很快上手协作起来，截止目前共参与了27期的内容，3次作为主负责人整理并发布，真心觉得在此过程中收获成长了很多，最大的感悟就是眼界不再局限于自己所认知或擅长的那一片领域内，改掉了一些不好的习惯(比如把知识放进收藏夹里吃灰)，在不断分享知识过程中对某些问题的理解也更加深刻广泛。
## 兔年寄语
1. 新的一年，希望在有精力前提下更多参与负责周刊的内容搜集整理，不断开拓自己知识领域；2. 顺利完成自己的课题，在投的几篇paper能顺利accpet，同时提升自己coding水平 ； 3. 互相协作半年之久，希望后续有机会和几位小伙伴线上/下见面有更多的交流认识。

**Comment by NiEntropy** [comment]
回望：
很高兴加入Openibox社群，在生信爱好者周刊小分队里！每一次参与周刊内容的分享都可以学到新的知识，接触新领域，给枯燥的科研生活注入活力！
兔年寄语：
 新年快乐！祝大家在新的一年里顺顺利利，事业有成，阖家团圆！
